x/staking/client/cli/query.go

```diff
@@ -163,6 +163,11 @@ $ %s query staking unbonding-delegations-from cosmosvaloper1gghjut3ccd8ay0zduzj6
 			return cliCtx.PrintOutput(ubds)
 		},
 	}
+
+	cmd.Flags().Int(flags.FlagPage, 0, "pagination page of unbonding delegations to query for")
```

Comment:
Were these changes tested? IIRC, pagination is supposed to start at 1.

Reply:
This was something I did on purpose not to break the current api, if not flag is set it will behave as before.

https://github.com/cosmos/cosmos-sdk/pull/6163/files#diff-dd5237cf8e7e5c9b6903247bab8e3173

What are your thoughts about it?

Reply:
Ahhh I see, meaning if you provide `0`, you get the entire set in the response?

Reply:
Yes, exactly, so people that used this on the past and it worked for them continue to work without any issues. I don't know, maybe is not needed.

Reply:
I think it'd be better if we support pagination and require a valid page to avoid maintaining hacks like this. Clients should now be expected to provide a valid page. If they want all results, they can simply provide a large limit.

Reply:
Perfect!